src/core/tsi/ssl_transport_security.c

```diff
@@ -44,6 +44,9 @@
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
+#if defined(OPENSSL_IS_BORINGSSL)
```

Comment:
Unfortunately boringssl reorganized openssl's code and mem.h is only available in boringssl's fork.